src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ModifyRolesForUserTest.java

```diff
@@ -95,7 +95,7 @@ public RequestSpecification generateBearerTokenForNonPuiManager() {
         }
     }
 
-    @Ignore// Ignoring until OpenId can be tested without hacking config file to point to p.r.
+    //@Ignore// Ignoring until OpenId can be tested without hacking config file to point to p.r.

```

Comment:
I thought this was being ignored "until OpenId can be tested without hacking config file to point to p.r." - has this now been resolved?